tutorials/SingleQubitGates/SingleQubitGates.ipynb

```diff
@@ -0,0 +1,874 @@
+    "= \\frac{1}{2}\\big(|0\\rangle\\langle0| - |0\\rangle\\langle0| + |0\\rangle\\langle1| + |0\\rangle\\langle1| + |1\\rangle\\langle0| + |1\\rangle\\langle0| + |1\\rangle\\langle1| - |1\\rangle\\langle1|\\big) =\n",
+    "\\frac{1}{2}\\big(2|0\\rangle\\langle1| + 2|1\\rangle\\langle0|\\big) = |0\\rangle\\langle1| + |1\\rangle\\langle0|$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Important Gates\n",
+    "\n",
+    "This section introduces some of the common single qubit gates, including their matrix form, their ket-bra decomposition, and a brief summary of what they do to various qubit states.\n",
+    "\n",
+    "This section relies on the following notation:\n",
+    "\n",
+    "$$|+\\rangle = \\frac{1}{\\sqrt{2}}\\big(|0\\rangle + |1\\rangle\\big) \\\\\n",
+    "|-\\rangle = \\frac{1}{\\sqrt{2}}\\big(|0\\rangle - |1\\rangle\\big) \\\\\n",
+    "|i\\rangle = \\frac{1}{\\sqrt{2}}\\big(|0\\rangle + i|1\\rangle\\big) \\\\\n",
+    "|-i\\rangle = \\frac{1}{\\sqrt{2}}\\big(|0\\rangle - i|1\\rangle\\big) \\\\\n",
+    "|\\psi\\rangle = \\alpha|0\\rangle + \\beta|1\\rangle$$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pauli Gates\n",
+    "\n",
+    "The Pauli gates, named after Wolfgang Pauli ([Wikipedia](https://en.wikipedia.org/wiki/Wolfgang_Pauli)), are based on the so-called **Pauli matrices**, which also have some special significance to the topic of measurement. All three Pauli gates are **self-adjoint**, meaning that each one is its own inverse.\n",
+    "\n",
+    "### Pauli $X$ Gate\n",
+    "\n",
+    "The $X$ gate is probably the most common quantum gate. It has the effect of \"inverting\" the state of the qubit it's applied to, swapping the amplitudes of the $|0\\rangle$ and $|1\\rangle$ states. It is sometimes referred to as the **bit flip** gate, or the **NOT** gate, because it acts like the classical NOT gate on the computational basis.\n",
+    "\n",
+    "### Pauli $Y$ Gate\n",
+    "\n",
+    "The $Y$ gate is somewhat similar to the $X$ gate, but it introduces a complex component into the mix.\n",
+    "\n",
+    "### Pauli $Z$ Gate\n",
+    "\n",
+    "The final Pauli gate is the $Z$ gate, sometimes known as the **phase flip** gate. It doesn't affect state $|0\\rangle$, and applies a phase of $-1$ to state $|1\\rangle$.\n",
+    "\n",
+    "<table style=\"border:1px solid\">\n",
+    "    <col width=50>\n",
+    "    <col width=50>\n",
+    "    <col width=150>\n",
+    "    <col width=200>\n",
+    "    <col width=150>\n",
+    "    <col width=50>\n",
+    "    <tr>\n",
+    "        <th style=\"text-align:center; border:1px solid\">Gate</th>\n",
+    "        <th style=\"text-align:center; border:1px solid\">Matrix</th>\n",
+    "        <th style=\"text-align:center; border:1px solid\">Ket-Bra</th>\n",
+    "        <th style=\"text-align:center; border:1px solid\">Applying to $|\\psi\\rangle$</th>\n",
```

Comment:
```suggestion
    "        <th style=\"text-align:center; border:1px solid\">Applying to $|\\psi\\rangle = \\alpha|0\\rangle + \\beta|1\\rangle$</th>\n",
```
By the way, if we do a matching change to the header of this column in each table, we won't need a row with the definition of \psi in the table with the list of basis gates - this will look more neat.